TF_Service_dotNet/TouchFree/Connections/ClientConnection.cs

```diff
@@ -178,52 +194,22 @@ internal void OnMessage(string _message)
 
             // We don't handle after-the-fact Handshake Requests here. We may wish to
             // if / when we anticipate externals building their own Tooling clients.
-
-            switch (action)
+            var queueHandler = messageQueueHandlers.SingleOrDefault(x => x.ActionCodes.Contains(action));
+            if (queueHandler != null)
+            {
+                queueHandler.AddItemToQueue(new IncomingRequest(action, null, content));
+            }
+            else if (action.ExpectedToBeHandled())
+            {
+                TouchFreeLog.ErrorWriteLine("Expected to be able to handle a " + action + " action but unable to find queue.");
```

Comment:
I am happy to move string manipulation to use interpolation, I don't want to spend too much time rewriting stuff though. It might be worth us putting together some linting and doing a big tech debt piece to cover items like this

Reply:
Added a techdebt task for this to cover other areas where we want to do this